src/test/java/seedu/address/logic/commands/misccommands/MiscCommandTestUtil.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.logic.commands.misccommands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+
+/**
+ * Contains helper functions for testing {@code MiscCommand}.
+ */
+public class MiscCommandTestUtil {
+
+    /**
+     * Executes the given {@code command}, confirms that <br>
+     * - the returned {@link CommandResult} matches {@code expectedCommandResult} <br>
+     * - the {@code actualModel} matches {@code expectedModel}
+     */
```

Comment:
Could I check if this is the correct syntax for the javadocs header?

Reply:
On a side note, I think this kind of Javadocs header is quite common in other parts of the `AB3` codebase.

Reply:
In that case I don't think we have to change it until a later point, where we should specifically target these header types. Otherwise LGTM!👌